src/adapters/pulsepointLite.js

```diff
@@ -34,13 +41,18 @@ function PulsePointLiteAdapter() {
       cn: 1,
       ca: 'BID',
       tl: 1,
-      'if': 0,
+      if: 0,
```

Comment:
This is interesting, i guess if could be reserved here? Will test to build and see if it still runs otherwise I will submit a issue to the prettier team